src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -1,131 +1,92 @@
-                            {% else %}
-                                <span id="resume-url">{{ profile.url }}</span>
-                            {% endif %}
-                            <span class="date">
-                                <em>by {{ profile.ip }}</em> at <em>{{ profile.time|date('r') }}</em>
-                            </span>
-                        </div>
-                    {% endif %}
+    {{ include('@WebProfiler/Profiler/header.html.twig', with_context = false) }}
 
-                    <div id="collector-content">
-                        {% include '@WebProfiler/Profiler/base_js.html.twig' %}
-                        {% block panel '' %}
-                    </div>
+    {% if tokens is defined %}
+        <div id="summary">
+            <div class="status status-undefined">
+                <div class="container">
+                    <p class="lead">Profile Search</p>
                 </div>
-                <div id="navigation">
-                    {% if templates is defined %}
-                        <ul id="menu-profiler">
-                            {% for name, template in templates %}
-                                {% set menu %}{{ template.renderBlock('menu', { 'collector': profile.getcollector(name)}) }}{% endset %}
-                                {% if menu != '' %}
-                                    <li class="{{ name }}{% if name == panel %} selected{% endif %}">
-                                        <a href="{{ path('_profiler', { 'token': token, 'panel': name }) }}">{{ menu|raw }}</a>
-                                    </li>
-                                {% endif %}
-                            {% endfor %}
-                            <li class="minimize">
-                                <a href="javascript:void(0);" title="Minimize toolbar" onclick="return toggleMenuPanels();">
-                                    <span class="label">
-                                        <span class="icon"><svg id="minimizePanelIcon" width="30" height="33" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 30 33" enable-background="new 0 0 30 33" xml:space="preserve"><path fill="#3F3F3F" d="M15 5C8.4 5 3 10.4 3 17c0 6.6 5.4 12 12 12s12-5.4 12-12C27 10.4 21.6 5 15 5z M19.1 21.5l-1.8 1.8L10.9 17 l6.3-6.3l1.8 1.8L14.6 17L19.1 21.5z"/></svg></span>
-                                        <strong>Minimize</strong>
-                                    </span>
-                                </a>
-                            </li>
-                        </ul>
-                    {% endif %}
-                    {{ render(path('_profiler_search_bar')) }}
-                    {% include '@WebProfiler/Profiler/admin.html.twig' with { 'token': token } only %}
+            </div>
+        </div>
+    {% elseif profile is defined %}
+        {% set status_code = profile.getcollector('request').statuscode|default(0) %}
```

Comment:
what if the request collector is not registered ? The template should account for it